lib/oli_web/pow/messages.ex

```diff
@@ -1,8 +1,9 @@
 defmodule OliWeb.Pow.Messages do
   @moduledoc """
-  Custom module that handles returned messages by pow actions.
+  Custom module that handles returned messages by pow actions for users.
```

Comment:
The `for users` specification is no longer necessary since it's being used for both users and authors :)